README.md

```diff
@@ -1 +1,23 @@
-# updated-tutorials
\ No newline at end of file
+# Updated Tutorials
+
+*Zuletzt geändert von SnowLeoo am 23.Juni 2020*
+---
+
+ Diese Repository ist gedacht für die alten Tutorials von [Zekro](https://www.youtube.com/user/Zekrommaster110). In dieser Repo werden in Ordnern die alten Tutorial Codes in neue umgeschrieben. Diese sind unterteil in jeweiligen Wrapper, Folge und der dazugehörige neue Code. 
```

Comment:
Würde ich ändern zu `Dieses Repository ist dazu gedacht, die alten Tutorials von [Zekro](https://www.youtube.com/user/Zekrommaster110) zu updaten`

Reply:
Auch noch:
wird in Ordnern :D